core/blockchain/impl/block_tree_impl.cpp

```diff
@@ -164,6 +185,15 @@ namespace kagome::blockchain {
     return outcome::success();
   }
 
+  outcome::result<void> BlockTreeImpl::addBlockBody(
+      primitives::BlockNumber block_number,
+      const primitives::BlockHash &block_hash,
+      const primitives::BlockBody &body) {
+    primitives::BlockData block_data{.hash = block_hash, .body = body};
```

Comment:
I believe that it is more canonical to create a separate constructor of primitives::BlockData

Reply:
We use C++17 ) This is common practice to use such constructions. IMO constructor is not needed